package.yaml

```diff
@@ -15,6 +15,7 @@ dependencies:
   - lens
   - megaparsec
   - mtl
+  - n-ary-functor
```

Comment:
wow, you used my n-ary-functor package! I'm so proud! but... where are you using it? I don't see any `import NAryFunctor`.